xbmc/Application.cpp

```diff
@@ -3047,9 +3047,12 @@ void CApplication::OnPlayBackStarted(const CFileItem &file)
 {
   CLog::LogF(LOGDEBUG,"CApplication::OnPlayBackStarted");
 
-  // check if VideoPlayer should set file item stream details from its current streams
-  if (file.GetProperty("get_stream_details_from_player").asBoolean())
+  /* Always update streamdetails, except for DVDs where we only update
+     streamdetails if total duration > 15m (Should yield more correct info) */
+  if (!(file.IsDiscImage() || file.IsDVDFile()) || m_appPlayer.GetTotalTime() > 15*60*1000)
```

Comment:
GetTotalTime is undefined / zero at OnPlayBackStarted

Reply:
@FernetMenta : Good point. Any idea how to proceed with this? Perhaps only drop the GetTotalTime()-part and replace it with the GetProperty() call ?

Reply:
For what reason do you place the code into OnPlaybackStarted?

Reply:
would you suggest `OnAVStarted()`? There all the required info should be ready, right?

Reply:
To me the intent described by the comment does not make much sense. Maybe some workaround to skip intro tracks instead of properly detecting the main title. Orignally introduced by @notspiff https://github.com/xbmc/xbmc/commit/8877348f00f15f51ff1955d39acfd4f21fb6050d
If you are not able to determine the original intent, it is better to drop the condition and save stream details always.

Reply:
Yes, makes sense. Changed it back to what I initially had.